iothub_client/tests/iothubmoduleclient_ll_ut/iothubmoduleclient_ll_ut.c

```diff
@@ -400,4 +411,70 @@ TEST_FUNCTION(IoTHubModuleClient_LL_SendMessageDisposition_Test)
+{
+    //arrange
+    STRICT_EXPECTED_CALL(IoTHubClientCore_LL_SendEventAsync(TEST_IOTHUB_CLIENT_CORE_LL_HANDLE, TEST_MESSAGE_HANDLE, TEST_EVENT_CONFIRMATION_CALLBACK, NULL));
+
+    //act
+    IOTHUB_CLIENT_RESULT result = IoTHubModuleClient_LL_SendTelemetryAsync(TEST_IOTHUB_MODULE_CLIENT_LL_HANDLE, TEST_MESSAGE_HANDLE, TEST_EVENT_CONFIRMATION_CALLBACK, NULL);
+
+    //assert
+    ASSERT_IS_TRUE(result == IOTHUB_CLIENT_OK);
+    ASSERT_ARE_EQUAL(char_ptr, umock_c_get_expected_calls(), umock_c_get_actual_calls());  
+}
+
+TEST_FUNCTION(IoTHubModuleClient_LL_SendPropertiesAsync_Test)
+{
+    //arrange
+    STRICT_EXPECTED_CALL(IoTHubClientCore_LL_SendReportedState(TEST_IOTHUB_CLIENT_CORE_LL_HANDLE, TEST_UNSIGNED_CHAR, TEST_SIZE_T, TEST_PROPERTY_ACKNOWLEDGED_CALLBACK, NULL));
+
+    //act
+    IOTHUB_CLIENT_RESULT result = IoTHubModuleClient_LL_SendPropertiesAsync(TEST_IOTHUB_MODULE_CLIENT_LL_HANDLE, TEST_UNSIGNED_CHAR, TEST_SIZE_T, TEST_PROPERTY_ACKNOWLEDGED_CALLBACK, NULL);
+
+    //assert
+    ASSERT_IS_TRUE(result == IOTHUB_CLIENT_OK);
+    ASSERT_ARE_EQUAL(char_ptr, umock_c_get_expected_calls(), umock_c_get_actual_calls());
+}
+
+TEST_FUNCTION(IoTHubModuleClient_LL_SubscribeToCommands_Test)
+{
+    //arrange
+    STRICT_EXPECTED_CALL(IoTHubClientCore_LL_SubscribeToCommands(TEST_IOTHUB_CLIENT_CORE_LL_HANDLE, TEST_COMMAND_CALLBACK, NULL));
+
+    //act
+    IOTHUB_CLIENT_RESULT result = IoTHubModuleClient_LL_SubscribeToCommands(TEST_IOTHUB_MODULE_CLIENT_LL_HANDLE, TEST_COMMAND_CALLBACK, NULL);
+
+    //assert
+    ASSERT_IS_TRUE(result == IOTHUB_CLIENT_OK);
+    ASSERT_ARE_EQUAL(char_ptr, umock_c_get_expected_calls(), umock_c_get_actual_calls());
+}
+
+TEST_FUNCTION(IoTHubModuleClient_LL_GetPropertiesAsync_Test)
+{
+    //arrange
+    STRICT_EXPECTED_CALL(IoTHubClientCore_LL_GetTwinAsync(TEST_IOTHUB_CLIENT_CORE_LL_HANDLE, (IOTHUB_CLIENT_DEVICE_TWIN_CALLBACK)TEST_PROPERTIES_RECEIVED_CALLBACK, NULL));
+
+    //act
+    IOTHUB_CLIENT_RESULT result = IoTHubModuleClient_LL_GetPropertiesAsync(TEST_IOTHUB_MODULE_CLIENT_LL_HANDLE, TEST_PROPERTIES_RECEIVED_CALLBACK, NULL);
+
+    //assert
+    ASSERT_IS_TRUE(result == IOTHUB_CLIENT_OK);
+    ASSERT_ARE_EQUAL(char_ptr, umock_c_get_expected_calls(), umock_c_get_actual_calls());
+
+}
+
+TEST_FUNCTION(IoTHubModuleClient_LL_GetPropertiesAndSubscribeToUpdatesAsync_Test)
+{
+    //arrange
+    STRICT_EXPECTED_CALL(IoTHubClientCore_LL_SetDeviceTwinCallback(TEST_IOTHUB_CLIENT_CORE_LL_HANDLE, (IOTHUB_CLIENT_DEVICE_TWIN_CALLBACK)TEST_PROPERTIES_RECEIVED_CALLBACK, NULL));
+
+    //act
+    IOTHUB_CLIENT_RESULT result = IoTHubModuleClient_LL_GetPropertiesAndSubscribeToUpdatesAsync(TEST_IOTHUB_MODULE_CLIENT_LL_HANDLE, TEST_PROPERTIES_RECEIVED_CALLBACK, NULL);
+
+    //assert
+    ASSERT_IS_TRUE(result == IOTHUB_CLIENT_OK);
+    ASSERT_ARE_EQUAL(char_ptr, umock_c_get_expected_calls(), umock_c_get_actual_calls());
+}
+
+END_TEST_SUITE(iothubmoduleclient_ll_ut)
```

Comment:
nit sorry: newline

Reply:
No need to apologize on nits!  Do I need a newline though?  I looked at the original file (including telling diff to NOT ignore blanks) and it looks the same as before my change.  Is the red minus sign on this an artifact of Git web diffing or is there something I'm missing?

Reply:
maybe. As long as the file has it its all good. I trust you haha